backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -222,6 +222,18 @@ public async Task<ActionResult<int>> UpdateLexEntryCount(string code)
         return result is null ? NotFound() : result;
     }
 
+    [HttpPost("updateAllLexEntryCounts")]
+    [AdminRequired]
+    public async Task<ActionResult> UpdateAllLexEntryCounts(bool onlyUnknown)
+    {
+        var projects = lexBoxDbContext.Projects.Where(p => p.Type == ProjectType.FLEx && (!onlyUnknown || p.FlexProjectMetadata == null)).ToArray();
+        foreach (var project in projects)
+        {
+            await projectService.UpdateLexEntryCount(project.Code);
```

Comment:
This will presumably be about 1500 requests. I'm guessing it would make sense to maybe space them out a bit? Or maybe add a `int take = 100` parameter to this endpoint?

Reply:
Decided to do both: 100 requests by default, and a pause of 10 milliseconds between each one. Which isn't very long for a human, but quite long for a computer; it should allow enough time for the server to handle other tasks in between update requests. And it will only add 1 second to the time the user (a dev, presumably). I also have it return the number of projects completed as you suggested, and show a notification popup so you can track that without having to have the JS console open. All that is in d56bc4210cac4fe8a31a748b1eeb931416d2fd5c and a50f7bf2f2e9b6eeb769e3432dfb57c46f3dfc96.